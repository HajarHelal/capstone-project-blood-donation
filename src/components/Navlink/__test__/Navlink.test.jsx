import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavLink from '../Navlink';

// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <NavLink
          path="/donate"
          styleName="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
          text="Donate"
          menuClose={() => console.log('Donate')}
        />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
