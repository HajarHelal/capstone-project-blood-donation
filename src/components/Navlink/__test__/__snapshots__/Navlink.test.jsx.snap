// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li>
  <a
    className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
    href="/donate"
    onClick={[Function]}
  >
    Donate
  </a>
</li>
`;
